packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/representationdata/services/RepresentationMetadataCreationService.java

```diff
@@ -14,30 +14,30 @@
 
 import java.util.Objects;
 
-import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.RepresentationData;
-import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.repositories.IRepresentationDataRepository;
-import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.services.api.IRepresentationDataCreationService;
+import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.RepresentationMetadata;
+import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.repositories.IRepresentationMetadataRepository;
+import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.services.api.IRepresentationMetadataCreationService;
 import org.eclipse.sirius.web.domain.services.IResult;
 import org.eclipse.sirius.web.domain.services.Success;
 import org.springframework.stereotype.Service;
 
 /**
- * Used to create new representation data.
+ * Used to create new representation metadata.
  *
  * @author sbegaudeau
  */
 @Service
-public class RepresentationDataCreationService implements IRepresentationDataCreationService {
+public class RepresentationMetadataCreationService implements IRepresentationMetadataCreationService {
 
-    private final IRepresentationDataRepository representationDataRepository;
+    private final IRepresentationMetadataRepository representationMetadataRepository;
 
-    public RepresentationDataCreationService(IRepresentationDataRepository representationDataRepository) {
-        this.representationDataRepository = Objects.requireNonNull(representationDataRepository);
+    public RepresentationMetadataCreationService(IRepresentationMetadataRepository representationMetadataRepository) {
+        this.representationMetadataRepository = Objects.requireNonNull(representationMetadataRepository);
     }
 
     @Override
-    public IResult<RepresentationData> create(RepresentationData representationData) {
-        this.representationDataRepository.save(representationData);
-        return new Success<>(representationData);
+    public IResult<RepresentationMetadata> create(RepresentationMetadata representationMetadata) {
```

Comment:
This feels equally odd for the same reasons

Reply:
Updated